File: Algorithms/26. Remove Duplicates from Sorted Array/golang/solution.go
func removeDuplicates(nums []int) int {
	var i int = 0
	if len(nums) == 0 {
		return i
	}

	for j := 1; j < len(nums); j++ {
		if nums[i] != nums[j] {
			i++
			nums[i] = nums[j]
		}
	}

	return i + 1
}

File: Algorithms/26. Remove Duplicates from Sorted Array/swift/solution.swift
class Solution {
    func removeDuplicates(_ nums: inout [Int]) -> Int {
        var i = 0
        guard nums.count > 0 else {
            return i
        }

        for j in 1..<nums.count {
            if nums[j] != nums[i] {
                i += 1
                nums[i] = nums[j]
            }
        }
        return i + 1
    }
}
